themes/mcf/styles/styles.scss

```diff
@@ -59,3 +67,31 @@ option {
     @apply underline;
   }
 }
+
+.footer__section {
+  > * {
+    @apply mb-4;
+  }
+  h5 {
+    @apply mb-4;
+  }
+}
+
+.custom-header,
+h1.custom-header,
+h2.custom-header {
+  font-family: "Greycliff", sans-serif;
```

Comment:
Let's have a catch up on how we can integrate the tailwind config values into our css here